SPECS/libconfig/libconfig.spec

```diff
@@ -1,64 +1,80 @@
+%global sover_major 11
+%global sover_minor 1
 Summary:        C/C++ configuration file library
 Name:           libconfig
-Version:        1.7.2
-Release:        2%{?dist}
+Version:        1.7.3
+Release:        1%{?dist}
 License:        LGPLv2+
-URL:            https://hyperrealm.github.io/libconfig/
-#Source0:       https://github.com/hyperrealm/%{name}/archive/v%{version}.tar.gz
-Source0:        %{name}-%{version}.tar.gz
-Group:          Development/Tools
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-
+Group:          Development/Tools
+URL:            https://hyperrealm.github.io/libconfig/
+Source0:        https://github.com/hyperrealm/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  autoconf
 BuildRequires:  texinfo
 
 %description
-Libconfig is a simple library for processing structured configuration files, like this one: test.cfg. This file format is more compact and more readable than XML. And unlike XML, it is type-aware, so it is not necessary to do string parsing in application code.
+Libconfig is a simple library for processing structured configuration files. This file format is more compact and more readable than XML. And unlike XML, it is type-aware, so it is not necessary to do string parsing in application code.
+
+%package        devel
+Summary:        Development libraries and headers for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    devel
+Development libraries and headers for %{name}
 
 %prep
-%setup -q
+%autosetup
 
 %build
-autoreconf
 %configure --disable-static
-make %{?_smp_mflags}
+%make_build
 
 %install
-make DESTDIR=%{buildroot} install
-rm -rf %{buildroot}%{_libdir}/*.la
+%make_install
+find %{buildroot} -type f -name "*.la" -delete -print
 rm -rf %{buildroot}%{_infodir}/dir
 
 %check
 cd ./tests
 ./libconfig_tests # This HAS to be run in the tests directory due to use of relative paths.
 cd ..
 
-%post -p /sbin/ldconfig
-
-%postun -p /sbin/ldconfig
+%ldconfig_scriptlets
 
 %files
-%license LICENSE
-%doc AUTHORS ChangeLog COPYING.LIB README
-%{_libdir}/libconfig*.so.*
+%license COPYING.LIB
+%doc AUTHORS ChangeLog README
+%{_libdir}/libconfig*.so.%{sover_major}
+%{_libdir}/libconfig*.so.%{sover_major}.%{sover_minor}*
```

Comment:
I think we shouldn't be restricting the minor version. This may cause unnecessary build breaks for non-breaking changes in the libraries. Usually breaking changes only increment the major version number.